lib/fluent/plugin/in_kubernetes_objects.rb

```diff
@@ -176,9 +176,7 @@ def start_watchers
 	resource_name = o.delete(:resource_name)
 	version = @storage.get(resource_name)
 	o[:resource_version] = version if version
-	@client.public_send("watch_#{resource_name}", o).tap { |watcher|
-	  create_watcher_thread resource_name, watcher
-	}
+        create_watcher_thread resource_name, o
```

Comment:
Is this extra space of indentation intentional?

Reply:
No it is not. I've fixed it.